.travis.yml

```diff
@@ -0,0 +1,7 @@
+language: objective-c
```

Comment:
I just noticed, the language is set to `objective-c`. Shouldn't that be `swift`. I'm not sure what the consequences are, though.